server/constants/constants.go

```diff
@@ -29,6 +29,14 @@ const (
 	Workitems          = "_workitems"
 	Edit               = "edit"
 
+	// Subscription constants
+	PublisherID      = "tfs"
+	ConsumerId       = "webHooks"
```

Comment:
This constants are not making sense consumerID is equal to webhooks and consumerActionID is http request ?
```suggestion
	ConsumerID       = "webHooks"
```